components/script/dom/formdata.rs

```diff
@@ -134,3 +139,60 @@ impl FormData {
         }
     }
 }
+
+pub fn generate_boundary() -> String {
```

Comment:
Not exactly. I found just guidelines but no spec documentation. So I looked at mozilla-central for inspiration